nasa/openmct
# Reimplementation import export json

This PR is extracted from https://github.com/nasa/openmct/pull/3816
The test is split to a separate PR: https://github.com/nasa/openmct/pull/4225

### All Submissions:


### Author Checklist


### Testing instructions
1. Export any domain objects by right clicking it and select 'export as JSON'
2. Right click a folder and import the objects back by selecting 'import from JSON'
3. The object should be imported and exported without any errors
4. The object that's been imported should be exactly the same as the one that's exported
